.gitignore

```diff
@@ -8,3 +8,5 @@
 /src/*.o
 /bats/
 /default-packages
+.idea
+*.un~
```

Comment:
What's this?

Reply:
Extending project's `.gitignore` with personal OS/tooling-related files and paths is usually not recommended practice.
Especially in this exact case, it is totally unrelated to the patch you are providing.

I suggest you configuring global `.gitignore` file for your workstation:
https://adamj.eu/tech/2022/11/06/git-set-up-a-global-ignore-file/

Reply:
I think IDEA and Vim are used widely enough to warrant adding.

It's okay to address additional concerns in a PR (as long as it's within reason, or course), but then each concern needs to be in its own commit (see the [contribution giude](https://github.com/pyenv/pyenv/blob/master/CONTRIBUTING.md)).